crates/biome_js_analyze/src/lint/style/use_filenaming_convention.rs

```diff
@@ -341,7 +341,7 @@ pub enum FileNamingConventionState {
 /// Rule's options.
 #[derive(Clone, Debug, Deserialize, Deserializable, Eq, PartialEq, Serialize)]
 #[cfg_attr(feature = "schemars", derive(JsonSchema))]
-#[serde(rename_all = "camelCase", deny_unknown_fields)]
+#[serde(rename_all = "camelCase", deny_unknown_fields, default)]
```

Comment:
`serde(default)` is already specified on every field.
You should use one of them, not both.